src/main/java/org/redisson/RedissonSetMultimap.java

```diff
@@ -69,6 +69,19 @@
                 Arrays.<Object>asList(getName()));
     }
 
+    public Future<Integer> keySizeAsync() {
+    	return commandExecutor.evalReadAsync(getName(), codec, RedisCommands.EVAL_INTEGER,
```

Comment:
Please replace it to `return commandExecutor.readAsync(getName(), LongCodec.INSTANCE, RedisCommands.HLEN, getName());` instead due to performance gain